### ISSUE TITLE
[NBug] Index was outside the bounds of the array.

### ISSUE THREAD

**Issue by emydex-shane-wild** [gitextensions/gitextensions#10788]
### Environment

- Git Extensions 4.0.2.16100
- Build 25100ec1f7c8da8f798613da74826038af81a50e
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.13
- DPI 96dpi (no scaling)

### Issue description

```
System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at GitExtensions.Plugins.BackgroundFetch.BackgroundFetchPlugin.<>c__DisplayClass11_0.<RecreateObservable>b__1(Int64 i)
   at System.Reactive.AnonymousSafeObserver`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/AnonymousSafeObserver.cs:line 43
   at System.Reactive.ObserveOnObserverLongRunning`1.Drain() in /_/Rx.NET/Source/src/System.Reactive/Internal/ScheduledObserver.cs:line 732
   at System.Reactive.ObserveOnObserverLongRunning`1.<>c.<.cctor>b__17_0(ObserveOnObserverLongRunning`1 self, ICancelable cancelable) in /_/Rx.NET/Source/src/System.Reactive/Internal/ScheduledObserver.cs:line 680
   at System.Reactive.Concurrency.NewThreadScheduler.<>c__DisplayClass7_0`1.<ScheduleLongRunning>b__0() in /_/Rx.NET/Source/src/System.Reactive/Concurrency/NewThreadScheduler.cs:line 94
   at System.Threading.Thread.StartHelper.Callback(Object state)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
```

I have set a 5 minute automated fetch. I wonder if it has something to do with the commands list in the options being empty?

### Steps to reproduce

Unsure

**Comment by gerhardol** [comment]
duplicate of #10700 
Fixed in #10702 

> I wonder if it has something to do with the commands list in the options being empty?

Yes, set it to `fetch --all`

**Comment by emydex-shane-wild** [comment]
> 

Many thanks @gerhardol
